tools/lsp/server/server.toit

```diff
@@ -53,9 +53,7 @@ CRASH-REPORT-RATE-LIMIT-MS ::= 30_000
 monitor Settings:
```

Comment:
Why is this a monitor?

Reply:
Because it blocks the settings access if a `request-settings` action is in process.

Reply:
It probably would be more common to just implement this with a mutex, but it makes sense.